packages/core/contracts/oracle/interfaces/OptimisticOracleInterface.sol

```diff
@@ -0,0 +1,245 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+pragma solidity ^0.6.0;
+pragma experimental ABIEncoderV2;
+
+import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+
+/**
+ * @title Financial contract facing Oracle interface.
+ * @dev Interface used by financial contracts to interact with the Oracle. Voters will use a different interface.
+ */
+abstract contract OptimisticOracleInterface {
```

Comment:
Should the OptimisticOracle inherit this interface?

Reply:
Probably! Will update.

Reply:
Done.